src/docs/content/docs/usage/sidecar_mode.md

```diff
@@ -0,0 +1,49 @@
++++
+[menu.docs]
+name = "Sidecar Mode"
+weight = 50
+identifier = "sidecar_mode"
+parent = "usage"
++++
+
+
+# Sidecar Mode
+
+Sidecar Mode is a way of deploying Cassandra Reaper with one reaper instance for each node in the cluster.
+The name "Sidecar" comes from [the Sidecar Pattern](https://github.com/microsoftdocs/architecture-center/blob/master/docs/patterns/sidecar.md) which describes a mechanism for co-locating an auxiliary service with its supported application.
+See also  [Design Patterns for Container-based Distributed Systems](https://www.usenix.org/conference/hotcloud16/workshop-program/presentation/burns).
+It is a pattern that is often used in Kubernetes, where the main application and the sidecar application are deployed as separate containers in the same pod.
+
+In Sidecar Mode, each Cassandra node process is deployed alongside a Reaper process; Cassandra is the parent application and Reaper is the sidecar.
+
+## Advantages
+
+ * *Security*: Each Reaper process only needs local JMX access. No need to configure remote JMX access and JMX authentication.
+ * *Fault Tolerance*: Since there is one Reaper process per node, the Reaper services will be as resilient as the parent Cassandra cluster.
```

Comment:
Kinda… 

If the Reaper process crashes then no repairs will run on that Cassandra node. This includes all token-ranges that have a replica on that node.